### PR TITLE
Fix a (potential) copy-paste error and (try to) re-enable the certificate tests

### PR DESCRIPTION
**Pull request by jean-edouard**
**What this PR does / why we need it**:
I found an odd assignment (`VirtHandlerServiceName = "virt-controller"`) in the code.
That constant is used for certificate objects, so it could have been causing the flakyness on the certificate tests.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jean-edouard** [comment]
/test pull-kubevirt-check-tests-for-flakes
A flake was found, but not related to the tests re-enabled by this PR...

File: pkg/virt-operator/creation/components/webhooks.go [hunk 1]
```diff
@@ -429,7 +429,7 @@ const VirtApiServiceName = "virt-api"
 
 const VirtControllerServiceName = "virt-controller"
 
-const VirtHandlerServiceName = "virt-controller"
+const VirtHandlerServiceName = "virt-handler"
```
Comment (davidvossel) [inline pkg/virt-operator/creation/components/webhooks.go:432]: whoa... I'm confused how this ever worked if we got the service name wrong.  Do you have a handle on what the implications of an issue like this?

Comment (jean-edouard) [reply pkg/virt-operator/creation/components/webhooks.go:432]: Nope, just randomly noticed that!

File: tests/infra_test.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ var _ = Describe("Infrastructure", func() {
 		})
 
 		// Flaky, randomly fails with timeout
```
Comment (dhiller) [inline tests/infra_test.go:82]: How about removing the comment also?

File: tests/infra_test.go [hunk 3]
```diff
@@ -145,7 +145,7 @@ var _ = Describe("Infrastructure", func() {
...
```
Comment (dhiller) [inline tests/infra_test.go:147]: Same here.

File: tests/infra_test.go (hunk 2 above)
Comment (jean-edouard) [reply tests/infra_test.go]: Thanks, fixed

File: tests/infra_test.go (hunk 3 above)
Comment (jean-edouard) [reply tests/infra_test.go]: Tx, fxd

**Review by dhiller — APPROVED** [review]
/lgtm

**Pull request merged**